### PR TITLE
Main Components

### PR DESCRIPTION
**Pull request by chansen424**
**Changes**
- **Login Page** proceeds to app without authentication.

- **Next and Upcoming Rides**

- **Ride Summary** after clicking on a particular ride

- Placeholder **Map** Image.

- **Start Ride Button** leads to Map
- **Navigation Bar** -- shouldn't change pages yet
- **Sample Data** allows us to see general layout _before_ connection to backend.
- Consolidated related widgets into single files.
- Set up basic app **theme**, including font import (might need to import more in the future to fully match design).
- Components sized based on device size.

**Concerns**
- What if display text, such as the rider's name, is really long? Widgets will need to adjust or change.

**Next Steps**
- Fix Display for Rider on Maps Page
- Google Authentication
- Maps

### CODE REVIEW

**Review by lsizemore8 — COMMENTED** [review]
Let's chat tomorrow with the designers before I approve it, but this looks good. Impressive work :)

File: lib/main.dart [hunk 1]
```diff
@@ -1,25 +1,19 @@
 import 'package:flutter/material.dart';
-import 'Home.dart';
+import 'package:firebase_auth/firebase_auth.dart';
```
Comment (lsizemore8) [inline lib/main.dart]: Let's remove all instances of the firebase_auth, since we won't be using firebase auth anymore.

Comment (chansen424) [reply lib/main.dart]: All fixed.

**Review by lsizemore8 — APPROVED** [review]
Good job! Go ahead and merge it

**Pull request merged**